scan-triggers/aws-python-bucket-full-and-scheduled-scan/template.yaml

```
@@ -289,7 +298,7 @@ Resources:
           };
 
           exports.lambda_handler = async (event) => {
-            console.log(event);
+            console.log(`[FilterFunction] event: ${util.inspect(event, {depth: null})}`);
```

```suggestion
            console.log('[FilterFunction] event: ', event);
```
No need for `util.inspect `